### PR TITLE
Multi store frontend tests ECI-150

### PR DESCRIPTION
**Pull request by will-in-wi**
Some refactoring to make tests multi-store aware, and the start of frontend tests.

This is all stuff that we felt fairly confident in already, so we don't have to make code changes.

### CODE REVIEW

File: devtools_m1/cypress/integration/CustomerCartInteractions.feature [hunk 1]
```diff
@@ -58,3 +58,27 @@ Feature: Customer Cart Interactions
...
+      And I add a 'simple' widget to my cart
+    Then A simple cart event should be sent to Drip
+    When I check out
+    Then A simple order event should be sent to Drip
+
+  Scenario: A customer adds a simple product to their cart when a different store is configured for Drip
+    Given I am logged into the admin interface
+      And I have set up a multi-store configuration
+      And I have configured Drip to be enabled for 'site1'
+      And I have configured a simple widget for 'main'
```
Comment (jachin) [inline devtools_m1/cypress/integration/CustomerCartInteractions.feature]: Maybe this is obvious with more context but `main` doesn't really tell me what this `simple widget` is configured for? Is is a store or a site? Or something else?

File: devtools_m1/cypress/integration/CustomerSubscriptionInteractions/steps.js [hunk 2]
```diff
@@ -82,7 +78,7 @@ When('I {string} from the general newsletter', function(state) {
   cy.log('Resetting mocks')
   cy.wrap(Mockclient.reset())
 
-  cy.visit('/newsletter/manage/')
+  cy.visit(getCurrentFrontendDomain() + '/newsletter/manage/')
```
Comment (jachin) [inline devtools_m1/cypress/integration/CustomerSubscriptionInteractions/steps.js]: It looks like you can call a function from a template string.
```suggestion
  cy.visit(`${getCurrentFrontendDomain()}/newsletter/manage/`)
```

Comment (will-in-wi) [reply devtools_m1/cypress/integration/CustomerSubscriptionInteractions/steps.js]: I wondered about that, but the syntax seemed to indicate not. Glad to find I was wrong, thanks!

File: devtools_m1/cypress/integration/CustomerCartInteractions.feature (hunk 1 above)
Comment (will-in-wi) [reply devtools_m1/cypress/integration/CustomerCartInteractions.feature]: Makes sense. I'll think about this one… In these tests, it's a store.

Comment (will-in-wi) [reply devtools_m1/cypress/integration/CustomerCartInteractions.feature]: I'm going to leave this as is for now and keep it in mind for a future refactor. I fear I'd end up in a rabbit hole trying to figure out decent naming, and I want to continue making progress. It's an easy change later.

**Pull request merged**